include/coap3/coap_session.h

```diff
@@ -410,7 +410,7 @@ coap_session_t *coap_session_get_by_peer(const coap_context_t *ctx,
    * outstanding CON message.
    * RFC 7252, Section 4.8 Default value of ACK_TIMEOUT is 2
    *
-   * Configurable using coap_session_set_ack_timeout()
+   * Configurable using coap_session_[gs]et_ack_timeout()
```

Comment:
Since configuration is done only via the _set_ variant of this function pair, I suggest omitting the `[gs]` set and phrase this comment as "Configurable using coap_session_set_ack_timeout()". This makes text-based searching ("grep") through header files easier.
The same holds for the following occurrences of `[gs]`.